libs/utils/src/lib/format/number.ts

```diff
@@ -90,7 +105,29 @@ export const formatNumber = (
   rawValue: string | number | BigNumber,
   formatDecimals = 0
 ) => {
-  return getNumberFormat(formatDecimals).format(Number(rawValue));
+  const decimalSeparator = getDecimalSeparator() || '.';
+  const groupSeparator = getGroupSeparator();
+  const groupSize = groupSeparator ? getGroupSize() : 0;
+  const formatted = new BigNumber(rawValue).toFormat(formatDecimals, {
+    decimalSeparator,
+    groupSeparator,
+    groupSize,
+  });
+  if (!formatDecimals) {
+    return formatted;
+  }
+  const minimumFractionDigits = Math.min(
+    Math.max(0, formatDecimals),
+    MIN_FRACTION_DIGITS
+  );
+  if (!minimumFractionDigits) {
+    return formatted;
+  }
+  const parts = formatted.split(decimalSeparator);
```

Comment:
Can you add a comment why this is needed, eg. By default we show all values with 2 dp